api/types.go

```diff
@@ -482,6 +483,7 @@ func DefaultOptions() Options {
 			UseMLock:  false,
 			UseMMap:   true,
 			UseNUMA:   false,
+			FlashAttn: false, // for CPU only compatibility
```

Comment:
Is there a downside to having this always enabled?

Reply:
I believe it won't work without either Metal or CUDA >=7 cards.

Reply:
I'm not sure this flag is used anymore given the code below – should we remove it?

Reply:
I left it so you could optionally disable it at runtime, happy to remove it if you think it's not useful though?

Reply:
I think we can remove it for this PR for simplicity. Also, the code below doesn't seem to use it:

https://github.com/ollama/ollama/pull/4120/files#diff-ca827c5e987ec2b99e6a9d31f0c107fd0b4057706665f23e786d0a8cee43dfa8R207

Reply:
Ah, I misunderstood how that worked, good call. Removed the changes to the API types.